presto-docs/src/main/sphinx/release/release-330.rst

```diff
@@ -0,0 +1,131 @@
+=========================
+Release 330 (XX Feb 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix incorrect behavior of :func:`format` for ``char`` values. Previously, the function
+  did not preserve trailing whitespace of the value being formatted. (:issue:`2629`)
+* Fix query failure in some cases when aggregation uses inputs from both sides of a join. (:issue:`2560`)
+* Fix query failures when dynamic filtering is on for certain multi-level joins. (:issue:`2659`)
```

Comment:
How about

> Fix query failure when dynamic filtering is enabled and the query contains certain multi-level joins.

Although I think "certain" could be phrased better

Reply:
use "complex" maybe ... I did not look at the specific queries causing the issues to be honest